src/pipelines/floodscan_pipeline.py

```diff
@@ -52,6 +54,9 @@ def _generate_raw_filename(self, date, type):
     def _generate_processed_filename(self, date):
         return f"aer_area_300s_v{date.strftime(DATE_FORMAT)}_v0{self.version}r01.tif"
 
+    def _generate_baseline_filename(self, date):
+        return f"baseline_v{date.strftime(DATE_FORMAT)}_v0{self.version}r01.nc4"
```

Comment:
I'm sure you've discussed with Zack, but just curious here why we're outputting netcdf? What's the reason for not keeping with the geotiff standard?

Reply:
yes i think COG is preferred. I'm not sure why we are using netcdf here @isatotun ?

Reply:
This has been from following up from Zack's code which retrieves from the historical netcdf file so I kept the format but if COG is preferred I can change this.